test/test_helper.exs

```diff
@@ -31,11 +31,13 @@ defmodule RollbarAPI do
   import Conn
 
   def start(pid) do
+    :timer.sleep(100)
```

Comment:
Moving this sleep to `stop/0` after `shutdown/1` call will be equivalent, also we keep sleeps in one place what feels more natural since slow shutdown is a reason for possible `start/1` failure.